recipes/xcfun/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set name = "xcfun" %}
+{% set version = "2.0.2" %}
+{% set sha256 = "b79a71861a9e1d0e33c1be89b34f6f052d430cd495a7df982c407ad0140c1dcd" %}
+{% set pyver = PY_VER.replace(".", "") %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/dftlibs/xcfun/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+  binary_relocation: true
+  run_exports:
+    - {{ pin_subpackage('xcfun', max_pin='x') }}
+
+requirements:
+  build:
+    - cmake >=3.11
+    - {{ compiler('cxx') }}
+    - pybind11 =*=*py{{ pyver }}*
+    - ninja
+  host:
+    - python
+  run:
+    - numpy
```

Comment:
Does this link against numpy or just use it at runtime?

Reply:
It just uses it at runtime